src/UrqlTypes.re

```diff
@@ -82,4 +82,21 @@ type operationResult = {
   data: operationData,
   [@bs.optional]
   error: UrqlCombinedError.t,
-};
\ No newline at end of file
+};
+
+/* The signature of the structure created by calling `.make()` on a `graphql_ppx` module. */
+type request('response) = {
+  .
+  "parse": Js.Json.t => 'response,
+  "query": string,
+  "variables": Js.Json.t,
+};
+
+/* The type of the handler function passed to Subscription and useSubscription.
+   `handler` corresponds to the type of the handler function _before_ the latest subscription has been parsed.
+   `parsedHandler` corresponds to the type of the handler function _after_ the latest subscription has been parsed. */
+type handler('acc) =
+  (~prevSubscriptions: 'acc, ~subscription: Js.Json.t) => 'acc;
```

Comment:
Shouldn't this be `option('acc)`?
```suggestion
  (~prevSubscriptions: option('acc), ~subscription: Js.Json.t) => 'acc;
```

Reply:
Yep, good catch! The way I understand `~prevSubscriptions`, it will _always_ be `undefined` the first time it runs, so wrapping it in `option` is correct 👍 Just missed this one.